tests/wpt/mozilla/tests/css/matchMedia.html

```diff
@@ -0,0 +1,188 @@
+                assert_true(mql instanceof MediaQueryList, "matchMedia(\"all\") returned MediaQueryList object.");
+            }, "window.matchMedia(\"all\")");
+
+            test(function(){
+                assert_idl_attribute(mql, "media", "Check that MediaQueryList.media exists.");
+            }, "MediaQueryList.media exists");
+
+            test(function(){
+                assert_readonly(mql, "media", "Check that MediaQueryList.media is readonly.");
+            }, "MediaQueryList.media is readonly");
+
+            test(function(){
+                assert_equals(mql.media, "all");
+            }, "MediaQueryList.media for \"all\"");
+
+            test(function(){
+                assert_idl_attribute(mql, "matches", "Check that MediaQueryList.matches exists.");
+            }, "MediaQueryList.matches exists");
+
+            test(function(){
+                assert_readonly(mql, "matches", "Check that MediaQueryList.matches is readonly.");
+            }, "MediaQueryList.matches is readonly");
+
+            test(function(){
+                assert_true(mql.matches);
+            }, "MediaQueryList.matches for \"all\"");
+
+            test(function(){
+                assert_inherits(mql, "addListener");
+            }, "MediaQueryList.addListener exists");
+
+            test(function(){
+                assert_true(mql.addListener instanceof Function);
+            }, "MediaQueryList.addListener is a Function");
+
+            test(function(){
+                assert_inherits(mql, "removeListener");
+            }, "MediaQueryList.removeListener exists");
+
+            test(function(){
+                assert_true(mql.removeListener instanceof Function);
+            }, "MediaQueryList.removeListener is a Function");
+
+            test(function(){
+                mql = window.matchMedia("::");
+                assert_true(mql instanceof MediaQueryList, "window.matchMedia(\"::\") returned MediaQueryList object.");
+                assert_equals(mql.media, "not all", "MediaQueryList.media serialized as \"not all\" from original string with syntax error.");
+            }, "MediaQueryList.media syntax error");
+
+            test(function(){
+                assert_false(mql.matches);
+            }, "MediaQueryList.matches for \"not all\"");
+
+            test(function(){
+                mql = iframe_window.matchMedia("(max-width: 199px), all and (min-width: 200px)");
+                assert_equals(mql.media, "(max-width: 199px), (min-width: 200px)");
+                assert_true(mql.matches);
+            }, "MediaQueryList.matches for \"(max-width: 199px), all and (min-width: 200px)\"")
+
+            test(function(){
+                mql = iframe_window.matchMedia("(min-aspect-ratio: 1/1)");
+                assert_true(mql.matches);
+            }, "MediaQueryList.matches for \"(min-aspect-ratio: 1/1)\"");
+
+            test(function(){
+                mql = iframe_window.matchMedia("(width: 200px)");
```

Comment:
It doesn't pass because we don't have support for that form https://github.com/servo/servo/blob/master/components/style/media_queries.rs#L28